### DISCUSSION TITLE
How do I change the board url?

### DISCUSSION THREAD

**Discussion by deepseawitch** [vichan-devel/vichan#823] [Q&A]
I want to rename a board (e.g. changing /a/ to /anime/ without losing any of the posts on it). Is there a way or is my only option moving the posts from the old board to a new one?. I am not able to change it from the mod panel and could not find anything on the documentation.

**Comment by perdedora** [comment]
There isn't an easy way to do this
Read this: https://github.com/vichan-devel/vichan/issues/446
